composer.json

```diff
@@ -27,18 +27,18 @@
     }
   },
   "require": {
-    "php": "^7.4",
-    "typo3/cms-backend": "^10.4 || ^11.5",
-    "typo3/cms-core": "^10.4 || ^11.5",
-    "typo3/cms-extbase": "^10.4 || ^11.5",
-    "typo3/cms-extensionmanager": "^10.4 || ^11.5",
-    "typo3/cms-filelist": "^10.4 || ^11.5",
-    "typo3/cms-fluid": "^10.4 || ^11.5",
-    "typo3/cms-scheduler": "^10.4 || ^11.5",
+    "php": "^7.4 || ^8.3",
+    "typo3/cms-backend": "^11.5",
+    "typo3/cms-core": "^11.5",
+    "typo3/cms-extbase": "^11.5",
+    "typo3/cms-extensionmanager": "^11.5",
+    "typo3/cms-filelist": "^11.5",
+    "typo3/cms-fluid": "^11.5",
+    "typo3/cms-scheduler": "^11.5",
     "typo3fluid/fluid": "^2.7"
   },
   "require-dev": {
-    "friendsofphp/php-cs-fixer": "^3.2",
+    "friendsofphp/php-cs-fixer": "^3.58",
     "roave/security-advisories": "dev-latest",
     "typo3/cms-frontend": "^10.4 || ^11.5",
```

Comment:
@tstahn This and ff. should also be `"^11.5"` for the 2.0 release?

Reply:
@abeutel you're absolutely right, I dropped `^10.4` here too.